### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by listguy** [thundergolfer/automated-github-organization-invites#12]
Getting a 500 error and redirected to a blank page with a title 'Internal Server Error'  when attempting to add a user through the app.
I entered an access token with `admin:org, admin:org_hook` privileges.

**Comment by Fratorec** [comment]
Same problem

**Comment by thundergolfer** [comment]
Did you deploy it using the Heroku button? If so you'd be able to quickly check the logs and send back the stack trace for more information on what went wrong 👍

**Comment by thundergolfer** [comment]
From testing myself, I can reproduce and see these logs: 

```
2021-04-09T07:43:43.849424+00:00 app[web.1]: 2021-04-09 07:43:43 - Octokit::Forbidden - GET https://api.github.com/orgs/thundergolfer-forks/teams: 403 - Must have admin rights to Repository. // See: https://docs.github.com/rest/reference/teams#list-teams
```

**Comment by thundergolfer** [comment]
@listguy were you by chance entering in your own Github username into the box? 

Using my own `username` triggers the 500 error, but using other usernames succeeds. I just created a new account, `thundergolfer-two`, and confirmed that I could successfully invite that new account. 

![image](https://user-images.githubusercontent.com/12058921/114148432-15b31c00-995d-11eb-8144-c1d776fe739c.png)

If you confirm that avoiding your own username fixes things, I'll update this issue title to track the bug. Entering in your own username should not crash the server.

**Comment by listguy** [comment]
Hey @thundergolfer  thanks for responding!
Really liked your tool and it did served me well in the end. 40 users were able to join my organization.
About the error: I did check it on my user name before getting the `500`.

**Comment by thundergolfer** [comment]
Fixed in cbe459f707c69c93ca0aee16d465f029e3a1ef19
